modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectLayoutLocalServiceTest.java

```diff
@@ -156,8 +156,8 @@ private long _addObjectField() throws Exception {
 		ObjectField objectField = _objectFieldLocalService.addCustomObjectField(
 			TestPropsValues.getUserId(), 0,
 			_objectDefinition.getObjectDefinitionId(), false, false, null,
-			LocalizedMapUtil.getLocalizedMap(name), StringUtil.randomId(), true,
-			"String");
+			LocalizedMapUtil.getLocalizedMap(name), StringUtil.randomId(),
```

Comment:
Hey @mlan0 , can we have a test to cover the scenario when it's the default object layout?

Reply:
Done in the last commit.